src/pages/route/route.component.ts

```diff
@@ -30,13 +31,26 @@ export class RouteComponent {
     private routeService: RouteService, private vehicleService: VehicleService,
     private alertService: AlertService, private connection: ConnectivityService,
     private modalCtrl: ModalController, private favoriteRouteService: FavoriteRouteService,
-    private alertCtrl: AlertController) {
+    private toastCtrl: ToastController, private alertCtrl: AlertController) {
     this.routeId = parseInt(navParams.get('routeId'), 10);
     this.alerts = [];
     ga('set', 'page', '/route.html');
     ga('send', 'pageview');
   }
 
+  showToast(message: string): void {
+    if (this.toast) {
+      this.toast.dismissAll();
+    }
+
+  this.toast = this.toastCtrl.create({
```

Comment:
Do you mean it's too large of a tab indentation? Or should I change the indentation overall?